mucommander-core/src/main/java/com/mucommander/ui/notifier/NotifierProvider.java

```diff
@@ -98,6 +98,7 @@ public static void displayBackgroundNotification(final NotificationType notifica
                 return;
             }
 
+            DesktopManager.requestUserAttention();  // bounce app icon if supported by OS
```

Comment:
Is it a good place?

Reply:
yeah, this seems like a good place for this. I think this package might also be a good place for the other parts - it seems to be that all the changes here are related to notifications